dashboard/backend/index.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+const express = require('express');
```

Comment:
We should be consistent with string quoting (double or single). In my case I prefer double quotation.

Thinks like this are automatically validated with linters. So maybe it's a good opportunity to integrate  **eslint** in our JavaScript code.